src/styles/intent.styl

```diff
@@ -0,0 +1,52 @@
+@require 'settings/palette.styl'
+.app-wrapper
+    width 100%
+.app-content
+    height 100vh
```

Comment:
I'm always cautious with `vh`. Unless there's a clear benefit of using it instead of `100%`, using a percent value is safer.

Reply:
Ok, I don't know the difference between `100%` and `100vh` or `100vw`. I saw unpleasant behavior with the classical `100%` so I tried `100vh` that works for my needs.

Reply:
So `height: 100vh` is `100% of the viewport height`, ie. the whole window. `height: 100%` is `100% of the parent's height`, which can be a lot of different things.
But on iOS at least, `vh` seems unreliable. i haven't investigated too much, but:

> vh on iOS is reported to include the height of the bottom toolbar in the height calculation, and the width of the sidebar (bookmarks) in the vw width calculation.

https://caniuse.com/#feat=viewport-units

Most of the time, when you want an element to take `100%` of the height, it's parent already uses `100%` of the height, so using `%` instead of `vh` is safer.

Reply:
Not just iOS, Chrome mobile does this too. They kinda interpreted the W3C Specification differently than others… The "bug" was reported a lot and they said this was exactly what they wanted and had no intention what so ever to change this. So yeah, `vh` is a doomed unit but `vw` is cool though.